packages/starknet-snap/src/index.ts

```diff
@@ -354,11 +354,10 @@ export const onHomePage: OnHomePageHandler = async () => {
     }
 
     // default network is testnet
-    let network = STARKNET_SEPOLIA_TESTNET_NETWORK;
-
+    let network = STARKNET_MAINNET_NETWORK;
     if (
       state.currentNetwork &&
-      state.currentNetwork.chainId !== STARKNET_TESTNET_NETWORK.chainId
+      state.currentNetwork.chainId !== STARKNET_MAINNET_NETWORK.chainId
```

Comment:
this line is for detect if the user using a deprecated network, so we dont need to change it